### DISCUSSION TITLE
Updated URL for documentation

### DISCUSSION THREAD

**Discussion by moonso** [Clinical-Genomics/scout#2432] [Show and tell]
The URL for the documentation has changed from clinicalgenomics.se to https://clinical-genomics.github.io/scout . I've updated the pointer in the scout repo. Hope that is ok @northwestwitch @dnil

**Comment by northwestwitch** [comment]
Shouldn't we fix this link in scout readme?

**Reply by moonso** [reply]
Yes good!

**Reply by moonso** [reply]
I did a PR
